version/2_3.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+echo "================= Installing Python 2.3 ==================="
+
+wget "https://www.python.org/ftp/python/2.3.7/Python-2.3.7.tgz"
+tar -xf Python-2.3.7.tgz
```

Comment:
we need to remove this tarfile from all versions once the installation is complete. there should not be any redundant components on the image

Reply:
ok